### PR TITLE
fix(interaction): 修复开启复制后, 无法复制表格外的文字

### PR DESCRIPTION
**Pull request by lijinke666**
### 👀 PR includes

🐛 Bugfix

- [x] Solve the issue and close #0

### 📝 Description

只在焦点在表格区域时才触发 keydown 和 keyup 事件

1. 修复复制表格外任意文字, 依然触发表格复制事件, 导致无法复制文字的问题

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| Kapture 2022-02-24 at 15 40 15 | Kapture 2022-02-24 at 15 38 11 |

### 🔍 Self Check before Merge

- [ ] Add or update relevant Docs.
- [ ] Add or update relevant Demos.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/s2-core/__tests__/unit/interaction/event-controller-spec.ts [hunk 1]
```diff
@@ -321,12 +335,65 @@ describe('Interaction Event Controller Tests', () => {
...
+    window.dispatchEvent(
+      new KeyboardEvent('keydown', {
+        key: InteractionKeyboardKey.COPY,
+        metaKey: true,
+      }),
+    );
+    expect(copied).toHaveBeenCalledWith(MOCK_COPY_DATA);
+  });
+
+  test("should dont't trigger sheet copy event", () => {
```
Comment (lcx-seima) [inline packages/s2-core/__tests__/unit/interaction/event-controller-spec.ts:359]: typo?
should not

**Review by xingwanying — APPROVED** [review]
LG

**Pull request merged**